sdk/azcore/runtime/poller_test.go

```diff
@@ -102,7 +102,7 @@ func TestLocPollerSimple(t *testing.T) {
 	}
 	var respFromCtx *http.Response
 	ctxWithResp := WithCaptureResponse(context.Background(), &respFromCtx)
-	_, err = lro.PollUntilDone(ctxWithResp, &PollUntilDoneOptions{Frequency: time.Second})
+	_, err = lro.PollUntilDone(ctxWithResp, &PollUntilDoneOptions{Frequency: time.Millisecond})
```

Comment:
Do smaller frequencies break something?

Reply:
Really I'm wondering whether something will go wrong in recording playback, where I'll be trying `time.Nanosecond`. I suppose the worst thing that could happen would be a somewhat longer sleep on systems with coarser clock resolution.

Reply:
Ah I see, I misunderstood the question.  This is what I get for answering before my first cup of coffee.